Packages/Core/Editor/Generator/Generator.cs

```diff
@@ -56,8 +56,9 @@ public void Generate(string type, string baseWritePath, bool isEquatable, List<A
 
             // Recursively search for template files. TODO: Is there a better way to find and load templates?
             var templateSearchPath = Runtime.IsUnityAtomsRepo ?
-                Directory.GetParent(baseWritePath).FullName : // "Packages"
+                Directory.GetParent(Runtime.UnityAtomsCorePath).FullName : // "Packages"
```

Comment:
I would prefer if you do it like this instead:
`Directory.GetParent(Application.dataPath).Parent.FullName`
In my mind it's better if we can compute the path at runtime instead of relying on constants.